CMakeLists.txt

```diff
@@ -11,6 +11,40 @@ set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
 set(SRC_DIR ${PROJECT_SOURCE_DIR}/src)
 
+# For google test
+include(FetchContent)
```

Comment:
Either remove this or set it on WIN32 only.

Reply:
Add on WIN32 only